commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/search/CommerceOrderDisplayTerms.java

```diff
@@ -57,6 +57,15 @@
 	public static final String START_CREATE_DATE_YEAR =
 		START_CREATE_DATE + "Year";
 
+	public static String[] getDisplayTerms() {
+		return new String[] {
+			ADVANCE_STATUS, CHANNEL, COMMERCE_ACCOUNT_ID, END_CREATE_DATE,
+			END_CREATE_DATE_DAY, END_CREATE_DATE_MONTH, END_CREATE_DATE_YEAR,
+			ORDER_STATUS, START_CREATE_DATE, START_CREATE_DATE_DAY,
+			START_CREATE_DATE_MONTH, START_CREATE_DATE_YEAR
+		};
+	}
+
```

Comment:
This should just be an array constant -- `public static final String[] DISPLAY_TERMS` or something like that.

Reply:
Hey Ethan, I did it like this to avoid illegal forward reference issue

Reply:
Ah ok, that does make sense. It makes more sense to me to just name it something that will come after all the constants it uses, to avoid that issue while still defining it as a `static final`. The whole class is `XDisplayTerms` so calling it `DISPLAY_TERMS` is a little redundant anyway. Maybe `VALID_TERMS`? Should we be including the terms defined in the base class, too?

Reply:
I think that, since this is internal class, we can leave it like this instead of refactoring all keys. @marco-leo what do you think?

Reply:
I wasn't suggesting refactoring all keys -- just naming this array `VALID_TERMS`... no refactoring necessary.